### PR TITLE
Added a new solution to a problem from 30daysofcode in Hackerrank

### PR DESCRIPTION
**Pull request by neha07kumari**

### CODE REVIEW

**Comment by neha07kumari** [comment]
Please review it. @Vishruth-S

File: Hackerrank_problems/Extra Long Factorials/solution.py [hunk 1]
```diff
@@ -0,0 +1,13 @@
+import math
+import os
+import random
+import re
+import sys
+
+# Complete the extraLongFactorials function below.
+def extraLongFactorials(n):     
+
```
Comment (Vishruth-S) [inline Hackerrank_problems/Extra Long Factorials/solution.py:9]: This seems to be incomplete. Could you please check

**Review by Vishruth-S — CHANGES_REQUESTED** [review]

**Comment by neha07kumari** [comment]
ˍAdded the new solution in C. please check it once more . @Vishruth-S

File: Hackerrank_problems/Extra Long Factorials/solution.py (hunk 1 above)
Comment (neha07kumari) [reply Hackerrank_problems/Extra Long Factorials/solution.py]: added the new solution in C .Please check it. @Vishruth-S

**Review by Vishruth-S — CHANGES_REQUESTED** [review]
The solution now works.
However please add some comments explaining the logic used :neutral_face: 😅

**Comment by neha07kumari** [comment]
Added the comments.

**Comment by Vishruth-S** [comment]
Already merged in #321 , closing this.

**Pull request closed without merging**